src/main/java/com/example/home_work_8/geometry/Circle.java

```diff
@@ -0,0 +1,40 @@
+package com.example.home_work_8.geometry;
+
+public class Circle implements Geometry {
+
+    final static double pi = Math.PI;
```

Comment:
There is no need to create this constant because you already have `Math.PI`, remove it.